src/Headers/Hydrodynamics.h

```diff
@@ -50,7 +51,7 @@ class EOS;
 
 class SimulationBase;
 
-static const FLOAT ghost_range = 2.5;
+static const FLOAT ghost_range = 3.0; //2.5;
```

Comment:
Travis was weirdly failing a test with the error norms, expecting 0.004 and getting 0.0040002 (or something like that).  I just hypothesised maybe it was a ghost issue and actually it 'solved' it (in the sense it was passing Travis).  But otherwise I don't know what was going on.  We can try again reducing ghost_range and see if we pass all the tests now (maybe there was a problem elsewhere that's now been fixed?)